src/Challenges.ts

```diff
@@ -52,7 +52,7 @@ export const getMaxChallenges = (i: number) => {
     if (i <= 15 && i > 10) {
         //Challenge 15 has no formal cap, so return 9001.
         if (i === 15) {
-            return 1
+            return 10
```

Comment:
this shouldn't need to be set; should be reverted to 0

Reply:
ok
It took me about an hour to get these working

Reply:
this should just handle display, ie sadistic ii used to display 0/0 completions.